### DISCUSSION TITLE
Saleor storefront and dashboard "Invalid Host header" when common.env has *

### DISCUSSION THREAD

**Discussion by marcorocci** [saleor/saleor#8537] [Deployment]
Hi,
details are here
[https://stackoverflow.com/questions/69886052/saleor-storefront-and-dashboard-invalid-host-header-when-common-env-has](https://stackoverflow.com/questions/69886052/saleor-storefront-and-dashboard-invalid-host-header-when-common-env-has)

someone can help me?

thanks in advance

**Comment by rakibulinux** [comment]
> Hi, details are here https://stackoverflow.com/questions/69886052/saleor-storefront-and-dashboard-invalid-host-header-when-common-env-has
> 
> someone can help me?
> 
> thanks in advance

Any solution for Invalid Host header?
